ivy/data_classes/array/experimental/statistical.py

```diff
@@ -180,6 +180,61 @@ def nanmean(
             self._data, axis=axis, keepdims=keepdims, dtype=dtype, out=out
         )
 
+    def nanmin(
+        self: ivy.Array,
+        /,
+        *,
+        axis: Optional[Union[Tuple[int], int]] = None,
+        out: Optional[ivy.Array] = None,
```

Comment:
would be better if `out` was the last argument of the function, given that's what we've done with all other functions, same applies to the other instance and static methods and backend implementations